pkg/storage/caching/caching.go

```diff
@@ -46,6 +48,16 @@ func (c *cachedOpenFGADatastore) ReadAuthorizationModel(ctx context.Context, sto
 	return model, nil
 }
 
+func (c *cachedOpenFGADatastore) FindLatestAuthorizationModelID(ctx context.Context, storeID string) (string, error) {
+	v, err, _ := c.G.Do(fmt.Sprintf("FindLatestAuthorizationModelID:%s", storeID), func() (interface{}, error) {
```

Comment:
If we were to use the `lookupGroup` for other cache purposes, we wouldn't need to think of collisions or adding another group. Happy to shorten the name tho.

Reply:
I think I'd prefer it be shortened, because it's uniquely bound to this instance of the `lookupGroup`. Only if we were to use it within the cachedOpenFGADatastore in another method would it be an issue. However, it's not a huge deal. Mostly a nit pick 😄